src/main/kotlin/com/gaelmarhic/quadrant/helpers/ActivityFilteringHelper.kt

```diff
@@ -56,6 +59,24 @@ class ActivityFilteringHelper(
                 )
             }
 
+    private fun List<ParsedManifest>.generateFullyQualifiedClassNames(): List<ParsedManifest> {
```

Comment:
> It does not "generate" a fully qualified class name all the time since some Activities are already declared that way

IMO it's fine, similar to List#clear which doesn't clear the list if it's already cleared but its output is always a cleared list

Reply:
Fair enough!